pkg/exchange/max/exchange.go

```diff
@@ -43,6 +44,55 @@ func (e *Exchange) Name() types.ExchangeName {
 	return types.ExchangeMax
 }
 
+func (e *Exchange) QueryTickers(ctx context.Context, symbol ...string) (map[string]types.Ticker, error) {
+	var ret = make(map[string]types.Ticker)
+
+	if len(symbol) == 1 {
+		ticker, err := e.client.PublicService.Ticker(strings.ToLower(symbol[0]))
+		if err != nil {
+			return nil, err
+		}
+		ret[strings.ToUpper(symbol[0])] = types.Ticker{
```

Comment:
could we use toGlobleSymbll function here?

Reply:
✔️ updated